Symfony/CS/Error/AbstractError.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Error;
+
+/**
+ * An abstraction of errors that can occur before and during fixing.
+ *
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+abstract class AbstractError
+{
+    const ERROR_TYPE_EXCEPTION = 1;
+    const ERROR_TYPE_LINT = 2;
```

Comment:
@keradus 

These constants are actually only ever used when reporting an error, how about using them for distinguishing errors which occurred before and after fixing?